README.md

```diff
@@ -495,6 +496,12 @@ node you render in downshift). Internally we use
 so if you use that package then you wont be adding any additional bytes to your
 bundle :)
 
+### suppressRefError
```

Comment:
This is excluded from the docs intentionally. It's a pretty far edge case so I don't think it's worth including in the docs.